.github/workflows/docs.yml

```diff
@@ -40,10 +40,6 @@ jobs:
       - name: Check No Changes
         run: git diff --exit-code --raw -p --stat
 
-      - name: Check Dead Links For All Files
-        if: github.ref == 'refs/heads/develop'
-        run: node docs/check-dead-links
```

Comment:
bye bye correct project documentation 😅

Reply:
Unfortunately the last dead link failures where mostly false positives that result in a "red" development branch even though there is no problem and especially is the problem not related to the latest merge to the develop branch. They had to be solved by adding an exception to the script.
The task has now become a manual step before releasing, so it no longer blames the latest "innocent" commit. 😉

Reply:
So this step should not really change to correct project documentation. 😉